apps/src/code-studio/pd/teachercon1819_registration/Teachercon1819Registration.jsx

```diff
@@ -5,7 +5,6 @@ import Joining from './Joining';
 import TravelPlans from './TravelPlans';
 import CoursePlans from './CoursePlans';
 import Releases from './Releases';
-import Confirmation from './Confirmation';
```

Comment:
Delete Confirmation.jsx too, since it's no longer being used.

Reply:
done :)